features/step_definitions/common_steps.rb

```diff
@@ -155,7 +167,7 @@
   puts "letter: #{@letter}"
 end
 
-Then /^I see the '(.*)' breadcrumb$/ do |breadcrumb_text|
+Then /^I see #{MAYBE_VAR} breadcrumb$/ do |breadcrumb_text|
```

Comment:
`MAYBE_VAR` is some magic thing that @risicle came up with which lets you either use a string value or a variable name in the scenario plain-english-y description.

So before we had a hardcoded breadcrumb value: 
```And I see the 'Platform as a Service' breadcrumb```

But now we have a variable.based one:
```And I see that lot.name breadcrumb```

`MAYBE_VAR` means that both versions will work.  (I think the word 'that' is the switch that decides whether we look for single quotes or do variable introspection.)